sys/key_value.js

```diff
@@ -194,6 +195,29 @@ class KVBucket {
             hyper.log('error/kv/putRevision', error);
             return { status: 400 };
         });
+
+        if (req.query.ignore_duplicates) {
+            return hyper.get({
+                uri: new URI([rp.domain, 'sys', 'key_value', rp.bucket, rp.key])
+            })
+            .then((oldContent) => {
+                if (stringify(req.body) === stringify(oldContent.body) &&
+                        (!req.headers['content-type']
+                        || req.headers['content-type'] === oldContent.headers['content-type'])) {
+                    hyper.metrics.increment(`sys_kv_${req.params.bucket}.unchanged_rev_render`);
+                    return {
+                        status: 200,
```

Comment:
According to https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.6, 204 (no content) might be more appropriate here.